test/jdk/java/foreign/security.policy

```diff
@@ -1,6 +1,7 @@
 grant codeBase "file:${test.classes}/*" {
     // Permissions needed to run the test
     permission java.util.PropertyPermission "os.name", "read";
+    permission java.util.PropertyPermission "os.arch", "read";
```

Comment:
This entry is not in alphabetical order, however the default `security.policy` content is same
https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/1455c242d55b982be6ff6effdfc27961f704a0f9/src/java.base/share/conf/security/java.policy#L25-L27